tickets/src/middlewares/walletMiddleware.js

```diff
@@ -82,6 +90,42 @@ const walletMiddleware = config => {
       }
     })
 
+    walletService.on(
+      'transaction.new',
+      (transactionHash, from, to, input, type, status) => {
+        // At this point we know that a wallet was found, because a new transaction
+        // cannot be created without it
+        dispatch(gotWallet())
```

Comment:
I am not sure what that is? This is used in `unlock-app` to hide the "look at your wallet" modal. Is this used here too? If so, I think the comment is not quite correct?